### PR TITLE
Add circuit documentation

### PR DESCRIPTION
**Pull request by kratochvil01**
I have added a section on device circuit.

I'm not sure where this section belongs. It is currently located in the API overview, but I think it is hard to find it here. I think it belongs thematically to the API, but maybe it belongs more in `index.md`, or maybe it deserves a separate file `circuit.md`. 

Maybe it doesn't make sense to create a circuit for `data_point`. In Evok 2 this entity was known as `unit_register` and it made sense to create a circuit based on the Modbus register address. But now this entity has a more general meaning and so it might make sense to change the creation of its circuit based on the variable name.

### CODE REVIEW

**Comment by tomashora** [comment]
- [ ] Navrh uvodniho textu: Circuit is a unique identifier of a particular entity (input, output, or function) of each device. The circuit is created automatically during initialization of Evok according to the table below.
- [ ] Do tabulky doplnit sloupecek s priklady, jak to muze vypadat
- [ ] V tabulce a v popisu pod tabulkou nahradit <count> za <number>... ale koukam, ze count je vsude v konfiguraci. Blby je ze count znamena počet, coz v anlictine nedava smysl a ma byt spravne number
- [ ] 1-wire example - ktery z formatu ma byt pouzit? s teckami nebo bez?

File: docs/apis.md [hunk 1]
```diff
@@ -38,3 +38,40 @@ The Webhook API provides a mechanism for pushing real-time updates to clients. E
...
+| Temp sensor         | *temp*         | <1-Wire_address\>                               | `2895DCD509000035`                | 
+| Non-volatile memory | *nvsave*       | <device_name\>                                  | `1`, `2`, `xS51`                  |
+| Modbus slave        | *modbus_slave* | <device_name\>                                  | `1`, `2`, `IAQ`                   |
+| Device info         | *device_info*  | <family_name\>\_<device_name\>_<serial_number\> | `Neuron_L533_0`, `Patron_S167_81` |
+| Data point          | *data_point*   | <device_name\>_<register_address\>              | `IAQ_0`, `IAQ_6`, `IAQ_10`        |
+| Modbus register     | *register*     | <device_name\>_<register_address\>              | `1_0`, `1_1`, `1_1000`            |
+
+- <device_name\>: Name defined in [Evok configuration].
+    - examples: `1`, `2`, `3`, `IAQ`, `xS51`, `xS18`.
+- <number\>: Sequence number (is based on the 'count' parameter in the [HW definition]).
```
Comment (cewbdex) [inline docs/apis.md:69]: What does it do? It's hard to understand.

**Review by cewbdex — CHANGES_REQUESTED** [review]

File: docs/apis.md (hunk 1 above)
Comment (kratochvil01) [reply docs/apis.md:69]: What you mean? It is legend for variables used in the table

File: docs/apis.md [hunk 2]
```diff
@@ -38,3 +38,40 @@ The Webhook API provides a mechanism for pushing real-time updates to clients. E
 ## RPC ([doc](./apis/rpc.md))
 
 The RPC (Remote Procedure Call) API is used for invoking procedures, functions or methods across a network.
+
```
[the remark this answers is not in the document]
Comment (kratochvil01) [reply docs/apis.md:41]: It says this in the PR description. The placement of this section is up for discussion.

File: docs/apis.md [hunk 3]
```diff
@@ -38,3 +38,40 @@ The Webhook API provides a mechanism for pushing real-time updates to clients. E
...
+# Device circuit
```
[the remark this answers is not in the document]
Comment (kratochvil01) [reply docs/apis.md:42]: This depends on the placement of this section. This has not been decided yet.

File: docs/apis.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kratochvil01) [reply docs/apis.md:41]: I don't think the configuration is the right place to put this. It's about accessing individual entities using the API. The way the circuits are created is not configurable.

**Pull request merged**